src/NuGet.Core/NuGet.Protocol.Core.Types/UserAgent.cs

```diff
@@ -22,10 +22,13 @@ public static void SetUserAgentString(UserAgentStringBuilder builder)
             }
 
             UserAgentString = builder.Build();
+            UserAgentVersion = builder.NuGetClientVersion;
         }
 
         public static string UserAgentString { get; private set; }
 
+        public static string UserAgentVersion { get; private set; }
```

Comment:
Use ``MinClientVersionUtility.GetNuGetClientVersion()`` instead.

This shouldn't be added here, it will just end up being a copy of the above property value which is cached already.